TSPL.docc/LanguageGuide/AccessControl.md

```diff
@@ -44,17 +45,36 @@ A *source file* is a single Swift source code file within a module
 Although it's common to define individual types in separate source files,
 a single source file can contain definitions for multiple types, functions, and so on.
 
+A *package* is a group of modules that you develop as a unit.
+You define the modules that form a package
+as part of configuring the build system you're using,
+not as part of your Swift source code.
+For example, if you use Swift Package Manager to build your code,
+you define a package in your `Package.swift` file
+using APIs from the [PackageDescription][] module;
```

Comment:
Is there a missing link for PackageDescription?

Reply:
This is a [shortcut reference link](https://spec.commonmark.org/0.30/#shortcut-reference-link) — the destination is defined at the end of the paragraph, on line 58.  Omitting text in the second set of square brackets indicates that the link should use the destination whose reference is the same as the text of the link.

I tend to format links in TSPL this way because it keeps the long URLs from interrupting the paragraph, instead putting the link's destination after the end of the paragraph that the link appears in.  Making the reference the same as the link text keeps the in-paragraph markup shorter, and removes the need to pick what short identifier to give to each URL (and the need to use a consistent style for those IDs).

Reply:
Thank you for the explanation! Makes perfect sense.